src/Nethermind/Nethermind.Core.Test/Builders/BlockTreeBuilder.cs

```diff
@@ -71,6 +73,7 @@ public BlockTreeBuilder OfChainLength(int chainLength, int splitVariant = 0, int
 
         private ISpecProvider _specProvider;
         private IEthereumEcdsa _ecdsa;
+        private Func<Block, Transaction, IEnumerable<LogEntry>> _logsForBlockBuilder;
```

Comment:
I appreciate since it must have been painful to work with this class, forgive the WTFs :)

Reply:
Its fine, just hard to name this well. Its for creating logs in a tree from the outside :)